vars/githubscm.groovy

```diff
@@ -42,36 +42,50 @@ def getRepositoryScm(String repository, String author, String branches) {
     return repositoryScm
 }
 
+def mergeSourceIntoTarget(String repository, String sourceAuthor, String sourceBranches, String targetAuthor, String targetBranches) { 
+    def repositoryUrl = "https://github.com/${sourceAuthor}/${repository}"
+    mergeRepo(repositoryUrl, sourceBranches, targetAuthor, targetBranches)
+}
```

Comment:
This is not used anywhere.

Reply:
is it not used by your pipeline?

Reply:
Nope, I use just githubscm.checkoutIfExists(..)

Reply:
Resolved this now with my commit.